.github/stale.yml

```diff
@@ -0,0 +1,12 @@
+daysUntilStale: 14
+daysUntilClose: 7
+exemptLabels:
+  - pinned
+  - security
+  - in backlog
+staleLabel: stale
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
```

Comment:
Thanks for the hint! I like the mention of the timespan and a way to tell the user how to keep it open. "Truly yours" is a bit too much for my taste. I adjusted it accordingly.